app/models/manageiq/providers/hawkular/inventory/persister/middleware_manager.rb

```diff
@@ -0,0 +1,44 @@
+module ManageIQ::Providers
+  class Hawkular::Inventory::Persister::MiddlewareManager < ManagerRefresh::Inventory::Persister
+    include ManagerRefresh::Inventory::Core
+
+    COMMON_ATTRIBUTES = %i(name ems_ref nativeid properties feed).freeze
+
+    has_inventory(
+      :model_class                 => provider_module::MiddlewareManager::MiddlewareDomain,
```

Comment:
Of course. I think all of them are middleware specific, so I'll place them in `middleware_manager.rb`.